share/lib/python/neuron/tests/utils/capture_stdout.py

```diff
@@ -0,0 +1,25 @@
+import sys, io, inspect, hashlib
+
+
+def capture_stdout(stmt, md5=False):
```

Comment:
Not sure, we should check that. The codecov checks postdate most of the work here, so I haven't (yet) spent any time looking into them...

Reply:
@alexsavulescu I'm wondering if this is because we use `pytest-cov` to collect Python coverage, but some Python-based tests are not driven by `pytest`? Do we have any examples of non-pytest Python tests?

Reply:
I'll try sprinkling some extra `coverage run` prefixes around in another PR to see if we can improve this.

Reply:
I started trying to address this in https://github.com/neuronsimulator/nrn/pull/2313, so I can resolve this now.